src/dodal/devices/positioner.py

```diff
@@ -0,0 +1,47 @@
+from typing import TypeVar
+
+from bluesky.protocols import Movable
+from ophyd_async.core import (
+    AsyncStatus,
+    StandardReadable,
+    StrictEnum,
+)
+from ophyd_async.core import StandardReadableFormat as Format
+from ophyd_async.epics.core import (
+    epics_signal_rw,
+)
+from ophyd_async.epics.motor import Motor
+
+T = TypeVar("T", bound=StrictEnum)
+
+
+class Positioner1D(StandardReadable, Movable[T]):
+    """1D stage with a enum table to select positions."""
+
+    def __init__(
+        self,
+        prefix: str,
+        datatype: type[StrictEnum],
+        positioner_pv_suffix: str = ":MP:SELECT",
+        name: str = "",
+    ) -> None:
+        self._stage_motion = Motor(prefix=prefix)
+        with self.add_children_as_readables(Format.CONFIG_SIGNAL):
```

Comment:
I think we want to do read to get the postion rather than the config?
```suggestion
        with self.add_children_as_readables():
```